src/components/base/JSearch/JSearch.js

```diff
@@ -39,6 +39,10 @@ class JSearch extends PureComponent<Props, ComponentState> {
     }
   }
 
+  componentWillUnmount() {
+    this.onChange('')
```

Comment:
Add FIXME comment here plz

Looks like it needs further investigation in the future